weave/trace/vals.py

```diff
@@ -753,6 +754,7 @@ def make_trace_obj(
     else:
         if hasattr(box_val, "ref"):
             setattr(box_val, "ref", new_ref)
+        initialize_object(box_val)
```

Comment:
Backup initialization path -- I wouldn't expect anything here, but included for safety